From 91153fd02e62f2eaca8e6c140baa4f2abf39c40e Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Wed, 30 Nov 2016 10:43:08 +0100
Subject: git: update to v2.11.0

Update to git version v2.11.0. Function write_archive()
dropped argument (int setup_prefix).
---
 ui-snapshot.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'ui-snapshot.c')

--- ui-snapshot.c.orig
+++ ui-snapshot.c
@@ -37,7 +37,7 @@ static int write_archive_type(const char *format, const char *hex, const char *p
 	/* argv_array guarantees a trailing NULL entry. */
 	memcpy(nargv, argv.argv, sizeof(char *) * (argv.argc + 1));
 
-	result = write_archive(argv.argc, nargv, NULL, 1, NULL, 0);
+	result = write_archive(argv.argc, nargv, NULL, NULL, 0);
 	argv_array_clear(&argv);
 	free(nargv);
 	return result;
